### DISCUSSION TITLE
Linq2db CLI Scaffolding getters and setters for a property

### DISCUSSION THREAD

**Discussion by saba-softiez** [linq2db/linq2db#3680] [Q&A]
Hi,
Thank you for building a great tool for the community.
In the T4 model generation process, I have used the below code to add custom getters and setters for a property. 
`column.GetBodyBuilders.Add(()=>{ return new List<string>(){"return base.Id;"}; });`

I would like to know how this can be achieved with CLI scaffolding tool with custom scaffold interceptors.

Thanks in advance,
Saba

**Comment by MaceWindu** [comment]
Currently that's not possible. I've created an issue to track it https://github.com/linq2db/linq2db/issues/3682

**Reply by saba-softiez** [reply]
Hi,
Thank you for your answer. 
I'm just curious to know if ICodeExpression properties can be introduced in the PropertyModel class for the Getter and Setter?
Similar implementations are available in the AssociationMetadata class. or it is too early to say?

Thanks.

**Reply by MaceWindu** [reply]
I would recommend to add examples of modifications you need to linked issue. It will help with extensibility API design
